README.md

```diff
@@ -1717,10 +1694,10 @@ After finishing all the settings in the GTM container, we click on "Send" in the
 We can go back to the Analytics tab, under the "Workspace" section and check the tag code termination. We see, in the upper right corner of the screen:
 `GTM-PGRSV4S`
```

Comment:
Suggestion: We see, in the upper right corner of the screen a code with the following structure: GTM-XXXXXXX

ID might change and also it isn't well to put ids in public docs.

Reply:
@tassioFront  done [1277799](https://github.com/juntossomosmais/frontend-guideline/pull/41/commits/1277799d6186b46764e821d83faa459c1da839ca)